src/vsr/replica.zig

```diff
@@ -8051,20 +8066,6 @@ pub fn ReplicaType(
             if (self.grid_repair_tables.available() > 0) {
                 self.sync_enqueue_tables();
             }
-
-            for (0..constants.clients_max) |entry_slot| {
```

Comment:
Maybe just leave a comment here, referencing that we are also syncing the client replies but that it is handled in `client_sessions_open_callback()` (to ensure that client sessions + replies are consistent).